packages/whenable/src/E.js

```diff
@@ -60,29 +59,16 @@ const makeEProxyHandler = (recipient, HandledPromise) =>
               );
             }
 
-            if (onSend && onSend.shouldBreakpoint(recipient, propertyKey)) {
-              // eslint-disable-next-line no-debugger
-              debugger; // LOOK UP THE STACK
-              // Stopped at a breakpoint on eventual-send of a method-call
-              // message,
-              // so that you can walk back on the stack to see how we came to
-              // make this eventual-send
-            }
```

Comment:
For all of these, what happens to this debuggability? Is it somehow delegated to the underlying `E`? If so, I'm confused because this E.js does not seem layered on top of the underlying one, but rather a whole cloth replacement. I don't want to lose the debuggability when programming with this higher level `E`.

Reply:
Seems fixed. Resolving.